test/WebJobs.Extensions.Tests/Extensions/MobileApps/MobileTableEndToEndTests.cs

```diff
@@ -242,6 +244,10 @@ private class MobileTableEndToEndFunctions
                 [MobileTable] out TodoItem[] arrayPoco,
                 [MobileTable] IAsyncCollector<TodoItem> asyncCollectorPoco,
                 [MobileTable] ICollector<TodoItem> collectorPoco,
+                [MobileTable(TableName = tableName)] out object newObject, // we explicitly allow object
+                [MobileTable(TableName = tableName)] out object[] arrayObject,
```

Comment:
Looks good. Do we need negative tests to verify invalid outputs: [MobileTable] out object newObject ?